js/src/modals/Transfer/Details/details.js

```diff
@@ -116,42 +116,44 @@ export default class Details extends Component {
     const { api } = this.context;
     const { balance, tag } = this.props;
 
-    const items = balance.tokens.map((balance, idx) => {
-      const token = balance.token;
-      const isEth = idx === 0;
-      let value = 0;
-
-      if (isEth) {
-        value = api.format.fromWei(balance.value).toFormat(3);
-      } else {
-        value = new BigNumber(balance.value).div(balance.token.format || 1).toFormat(3);
-      }
-
-      const label = (
-        <div className={ styles.token }>
-          <img src={ token.images.small } />
-          <div className={ styles.tokenname }>
-            { token.name }
+    const items = balance.tokens
+      .filter((token) => token.value.gt(0))
+      .map((balance, idx) => {
+        const token = balance.token;
+        const isEth = idx === 0;
```

Comment:
Well, yes, find(). However the way it is setup is that ETH is always in the first position. It needs to show up first where displayed, balance get pulled first, etc.